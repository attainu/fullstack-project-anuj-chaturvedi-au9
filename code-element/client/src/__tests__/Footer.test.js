
import React from 'react'
import { shallow, mount } from 'enzyme'
import Footer from '../components/layout/Footer'

     
    it('it renders Footer',()=>{
        const wrapper = shallow(<Footer/>);
        expect(wrapper).toMatchSnapshot();
        
    });

    

