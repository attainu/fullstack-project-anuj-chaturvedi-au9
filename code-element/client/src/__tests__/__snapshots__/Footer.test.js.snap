// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders Footer 1`] = `ShallowWrapper {}`;
